src/sweeper/__main__.py

```diff
@@ -4,60 +4,105 @@
 sweeper
 
 Usage:
-  sweeper sweep duplicates <table_name> --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path>]
-  sweeper sweep empties <table_name> --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path>]
-  sweeper sweep invalids <table_name> --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path>]
+  sweeper sweep duplicates --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]
+  sweeper sweep empties --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]
+  sweeper sweep invalids --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]
+  sweeper sweep all-checks --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]  
 
 Arguments:
-  workspace   - path to workspace eg: `c:\\my.gdb`
-  table_name  - name of feature class or table eg: `Roads`
+  workspace - path to workspace eg: `c:\\my.gdb`
+  table_name - name of feature class or table eg: `Roads`
   report_path - folder to save report to eg: `c:\\temp`
   backup_path - place to create a temp gdb and import original table
 
 Examples:
-  sweeper sweep empties thing --workspace c:\\data\\thing
+  sweeper sweep all-checks --workspace=c:\\data\\thing --save-report=c:\\temp --try-fix --backup-to=c:\\temp
 '''
 import os
 import sys
 
 from docopt import docopt
 
-from . import backup, report
+from . import backup, report, workspace_info
 from .sweepers.duplicates import DuplicateTest
 from .sweepers.empties import EmptyTest
 
 
 def main():
-    '''Main entry point for program. Parse arguments and pass to sweeper modules
+    '''Main entry point for program. Parse arguments and pass to sweeper modules.
     '''
-
     args = docopt(__doc__, version='1.0.0')
 
     #: backup input file before quality checks
     if args['--backup-to']:
-        backup.backup_data(os.path.join(args['--workspace'], args['<table_name>']), args['--backup-to'])
+        backup.backup_data(os.path.join(args['--workspace'], args['--table-name']), args['--backup-to'])
```

Comment:
this will need to backup an entire workspace if the table name is None right? I don't think I see that code anywhere.

Reply:
good point.  i'll code that in.

Reply:
this sounds like it's going to change based on the feedback from zach and erik about running on a workspace